### PR TITLE
KOGITO-8986: SWF Viewer - missing icon for parameters for Switch state

### PR DESCRIPTION
**Pull request by hasys**
Hello @LuboTerifaj, @romartin,

there is an icon for the Switch State. Only one of Data or Event are allowed by the spec, so I show by default Data. See [KOGITO-8986](https://issues.redhat.com/browse/KOGITO-8986) for more details.

image

### CODE REVIEW

File: packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-api/src/main/resources/org/kie/workbench/common/stunner/sw/resources/i18n/SWConstants.properties [hunk 1]
```diff
@@ -351,4 +351,12 @@ Eventfilter.null=EventDataFilter is not defined
 Eventfilter.parameter=EventDataFilter
 Eventfilter.usedata=useData
 Eventfilter.data=data
-Eventfilter.tostatedata=toStateData
\ No newline at end of file
+Eventfilter.tostatedata=toStateData
+
+Condition.null=Condition is not defined
+Condition.data=StateDataCondition
```
Comment (LuboTerifaj) [inline packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-api/src/main/resources/org/kie/workbench/common/stunner/sw/resources/i18n/SWConstants.properties]:
```suggestion
Condition.data=DataConditions
```

File: packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-api/src/main/resources/org/kie/workbench/common/stunner/sw/resources/i18n/SWConstants.properties [hunk 2]
```diff
@@ -351,4 +351,12 @@ Eventfilter.null=EventDataFilter is not defined
...
+Condition.dataname=Name
+Condition.datavalue=Condition
+Condition.event=EventDataCondition
```
Comment (LuboTerifaj) [inline packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-api/src/main/resources/org/kie/workbench/common/stunner/sw/resources/i18n/SWConstants.properties]:
```suggestion
Condition.event=EventConditions
```

**Review by LuboTerifaj — COMMENTED** [review]
Hello @hasys ,

I have just two inline comments. Would it make sense to use proposed changes?

Other than that, it looks good to me.

Thanks!

**Comment by hasys** [comment]
Changes are merged, thank you @LuboTerifaj !

**Review by LuboTerifaj — APPROVED** [review]
It looks good to me now. Thanks!

**Review by romartin — APPROVED** [review]

**Pull request merged**